### PR TITLE
minor typo fix and added extra security checks

### PR DESCRIPTION
**Pull request by bernardpeh**
added .gitignore and remove references to the "pet shop" tutorial. Added a check to not transfer token to ownself.

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ App = {
     var amount = parseInt($('#TTTransferAmount').val());
     var toAddress = $('#TTTransferAddress').val();
 
-    console.log('Transfer ' + amount + ' TT to ' + toAddress);
+    console.log('Transfering ' + amount + ' TT to ' + toAddress);
```
Comment (raid5) [inline src/js/app.js]: Typo, this should be `Transferring`.

File: src/js/app.js [hunk 2]
```diff
@@ -57,20 +57,30 @@ App = {
 
       var account = accounts[0];
 
+      // user cannot transfer tokens to himself
+      if (toAddress.toLowerCase() == account.toLowerCase()) {
+        alert('Sorry, you cannot transfer token to yourself');
```
Comment (raid5) [inline src/js/app.js]: I would reword these two lines to:

`Sorry, you cannot transfer tokens to yourself` (make token plural)

**Comment by bernardpeh** [comment]
fixed. nice tutorial.

File: src/js/app.js [hunk 3]
```diff
@@ -57,10 +57,10 @@ App = {
 
       var account = accounts[0];
 
-      // user cannot transfer tokens to himself
+      // cannot transfer tokens to ownself
       if (toAddress.toLowerCase() == account.toLowerCase()) {
-        alert('Sorry, you cannot transfer token to yourself');
-        throw new Error('cannot transfer token to yourself');
+        alert('Sorry, you cannot transfer tokens nto yourself');
```
Comment (raid5) [inline src/js/app.js]: typo: `nto` should be `to`
